azurerm/provider.go

```diff
@@ -297,8 +297,8 @@ func Provider() terraform.ResourceProvider {
 		"azurerm_hdinsight_storm_cluster":                            resourceArmHDInsightStormCluster(),
 		"azurerm_healthcare_service":                                 resourceArmHealthcareService(),
 		"azurerm_image":                                              resourceArmImage(),
-		"azurerm_iot_dps":                                            resourceArmIotDPS(),
-		"azurerm_iot_dps_certificate":                                resourceArmIotDPSCertificate(),
```

Comment:
So we typically leave in the old resources with a deprecation notice so there is no breaking changes